### ISSUE TITLE
No releases published despite the action running

### ISSUE THREAD

**Issue by feli-flitzberg** [nwhitehead/soundfont-godot#1]
In trying to download the release package for this repo, I noticed that while the Github Action to make the package was run, the release package itself is not available for download. Is there an extra step needed to make that package available?

**Comment by amberwhitehead** [comment]
@feli-flitzberg Yeah, I think there needs to be a publish to PyPI step in the actions. I think I have one from another project I can copy over. So far I've been doing it manually which is a pain.

**Comment by amberwhitehead** [comment]
Updating my previous comment: should be a publish to github releases in the workflow script (not PyPI). But this project is not ready for public use yet. I'm actually going a different direction with this now. Seeing how to do shared state management between threads in C++ for this project made me swear off C++ entirely a while ago. I switched to full Rust and like it much better. No more audio glitches and mysterious concurrency problems, yay. Godot and Rust actually work well together. That project is still being baked though.
